contrib/fast-hdfs-resource/src/main/java/org/apache/ambari/fast_hdfs_resource/Runner.java

```diff
@@ -127,6 +128,24 @@ public static void main(String[] args)
           }
 
           if (resource.getAction().equals("create")) {
+            if ("file".equals(resource.getType())) {
```

Comment:
This change makes fast-hdfs-resource not change permissions like owner and mode for existing file.